docs/specs/consensus-gate.spec.md

```
@@ -0,0 +1,261 @@
+
+11. **INV-011:** Every Vote MUST carry voter identity
+12. **INV-012:** Every Decision MUST record all Votes that were considered
+13. **INV-013:** When `adr_ref` is set, the ADR file MUST exist at that path
+
+---
+
+## 4. Decision Rules
+
+### 4.1 Tallying Logic
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)